src/App.js

```diff
@@ -159,7 +161,9 @@ class App extends Component {
       return {
         bytecode: this.state.compilationResult['bytecode'],
         bytecode_runtime: this.state.compilationResult['bytecode_runtime'],
-        ir: this.state.compilationResult['ir']
+        abi: JSON.stringify(this.state.compilationResult['abi']),
+        ir: this.state.compilationResult['ir'],
+        source_map_asm_list: this.state.compilationResult['source_map_asm_list']
```

Comment:
How to get `source_map_asm_list` ?
It seems that current version of vyper.live doesn't return it...

Reply:
I misunderstood that `vyper-serve` has already got to return that. I'll remove this.